go.mod

```diff
@@ -24,6 +24,8 @@ require (
 
 require go.etcd.io/bbolt v1.3.6
 
+require github.com/oliveagle/jsonpath v0.0.0-20180606110733-2e52cf6e6852 // indirect
```

Comment:
I don't see where I added this or used this? When I try to remove it brings it back when I mage build